ql/src/java/org/apache/hadoop/hive/ql/exec/DataSketchesFunctions.java

```diff
@@ -96,19 +97,31 @@ private DataSketchesFunctions() {
     return descriptors;
   }
 
+  public SketchFunctionDescriptor getSketchFunction(String className, String function) {
```

Comment:
Let's make className an `enum`, it will be neat.

Reply:
Yes it would be great - but I would also like to change the function to be an enum as well - I would like to postpone this to a later patch. I've opened: HIVE-23313